frontend/modules/admin_portal/students_view/AdminStudents.js

```diff
@@ -335,6 +332,7 @@ export default React.createClass({
     }
     if (index === -1){
       newStudentGrade.grades.push({assnId: assnId});
+      index = newStudentGrade.grades.length - 1;
```

Comment:
this line was added to fix a bug detected while submitting new grades in the class portal